### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by isaac-s**

### CODE REVIEW

File: content/agents/installation.md [hunk 1]
```diff
@@ -67,7 +67,9 @@ the download script is executed. It downloads the installation script
 from the temporary link and executes it.
 To use the `init_script` method, the IaaS provider and Cloudify plugin
 need to support userdata. Currently, the Openstack
-and AWS plugins support this installation method.
+and AWS plugins support this installation method. In addition, the image in
+use must already have Python 2.7.x installed and available through the
+system's `PATH`.
```
Comment (EarthmanT) [inline content/agents/installation.md]: What about GCP?

Comment (isaac-s) [reply content/agents/installation.md]: @EarthmanT I just checked and it seems that GCP also supports it. Can you confirm?

Comment (bmansheim) [reply content/agents/installation.md]: @EarthmanT What do you say, Trammell?

Comment (EarthmanT) [reply content/agents/installation.md]: GCP and Azure support it.

Comment (bmansheim) [reply content/agents/installation.md]: @isaac-s Please update accordingly.

**Pull request merged**